platform/mds/mds-web/src/main/resources/webapp/js/controllers.js

```diff
@@ -2001,24 +2002,28 @@
         * Adds a new index and sets it as the active one
         */
         $scope.addNewIndex = function () {
-            var newLookup = {
-                lookupName: $scope.getLookupNameForNewLookup(),
-                singleObjectReturn: true,
-                indexRequired: true,
-                lookupFields: []
-            };
+            if($scope.notDisabled === true) {
```

Comment:
1. if ($scope.notDisabled)
2. why you need this condition ?  is there a possibilty that it will be false here ?